views/js/qtiCreator/tpl/forms/item.tpl

```diff
@@ -35,3 +35,20 @@
         {{__ "Define whether the item should be time dependent on delivery."}}
     </span>
 </div>
+
+{{#if languagesList}}
+    <div class="panel">
+        <label for="xml:lang">
+            {{__ "Language"}}
+        </label>
+        <span class="icon-help tooltipstered" data-tooltip="~ .tooltip-content:first" data-tooltip-theme="info"></span>
+        <span class="tooltip-content">
+            {{__ "Define item language."}}
+        </span>
+        <select name="xml:lang">
+            {{#each languagesList}}
+                <option value="{{@key}}"{{#equal @key ../xml:lang}} selected="selected"{{/equal}}>{{this}}</option>
+            {{/each}}
+        </select>
```

Comment:
sorry missed that point during the first review but you need to use the select2 here to make it visually consistent with the other properties like in https://github.com/oat-sa/extension-tao-itemqti/blob/3455ccc0304dd4485686c7870762f5307c340765/views/js/qtiCreator/tpl/forms/interactions/textEntry.tpl#L23-L27

Reply:
Changed. Missed that point that inside backend we use select2 lib too, not only during delivery